.gitignore

```diff
@@ -12,3 +12,5 @@ phpunit.xml
 
 ImageMagick*.tar.gz
 /ImageMagick-*/
+
+.idea
```

Comment:
This should not be versioned. Instead, you can use a global ignore file.

Example of command to create such file:

```
git config --global core.excludesfile ~/.gitignore
touch ~/.gitignore
echo ".idea" >> .gitignore
```

This will create a global ignore file that you can use to remove all unnecessary files from Git, like `.DS_STORE` (for macs), or `thumbs.db` for Windows, etc.

Reply:
Thanks for that tip :)